### PR TITLE
Alphafold 2.3.0

### PR DESCRIPTION
**Pull request by henrywotton**
Now support the latest version of AlphaFold: AlpaFold 2.3.0

### CODE REVIEW

File: alphapulldown/create_individual_features.py [hunk 1]
```diff
@@ -124,13 +135,13 @@ def create_global_arguments(flags_dict):
...
 
     # Path to the Uniclust30 database for use by HHblits.
-    if FLAGS.uniclust30_database_path is None:
-        uniclust30_database_path = os.path.join(
-            FLAGS.data_dir, "uniclust30", "uniclust30_2018_08", "uniclust30_2018_08"
-        )
-    else:
-        uniclust30_database_path = FLAGS.uniclust30_database_path
-    flags_dict.update({"uniclust30_database_path": uniclust30_database_path})
+    # if FLAGS.uniclust30_database_path is None:
```
Comment (jkosinski) [inline alphapulldown/create_individual_features.py]: Remove the commented code?

File: alphapulldown/create_individual_features.py [hunk 2]
```diff
@@ -78,12 +79,22 @@ def create_global_arguments(flags_dict):
     global mgnify_database_path
     global bfd_database_path
     global small_bfd_database_path
-    global uniclust30_database_path
+    # global uniclust30_database_path
```
Comment (jkosinski) [inline alphapulldown/create_individual_features.py]: Remove the commented code?

**Review by henrywotton — COMMENTED** [review]
Now allow the user to five specific uniref30 database

**Review by jkosinski — APPROVED** [review]
Looks good. The flags management in the create_individual_features.py is very verbose and unnecessary but we can fix it later.

**Pull request merged**